### PR TITLE
feat: Add resource_reports field in Table API ( Atlas proxy)

### PR DESCRIPTION
**Pull request by dechoma**
### Summary of Changes

Add resource_reports field in Table API ( atlas proxy)  to support integration with tools generating static HTML reports, such as [popmon](https://github.com/ing-bank/popmon), [pandas-profiling]( https://github.com/pandas-profiling/pandas-profiling) or [great_expectations](https://github.com/great-expectations/great_expectations)..

### Tests

Updated existing unit tests

### Documentation

N/A

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`

Resolve lyft/amundsen#525

### CODE REVIEW

**Comment by mgorsk1** [comment]
cc @feng-tao @verdan

File: metadata_service/proxy/atlas_proxy.py [hunk 1]
```diff
@@ -377,6 +402,8 @@ def get_table(self, *, table_uri: str) -> Table:
 
             columns = self._serialize_columns(entity=entity)
 
+            reports_guids = [report.get("guid") for report in attrs.get("reports") or list()]
```
Comment (verdan) [inline metadata_service/proxy/atlas_proxy.py:405]: @dechoma is it possible to filter out the active reports here already ? I think you'll have the active status available here, so no need to make a query with the inactive queries.

Comment (dechoma) [reply metadata_service/proxy/atlas_proxy.py:407]: Unfortunately, attributes contain only report typeName and guid`{'owner': 'testuser', 'temporary': False, 'reports': [{'guid': '50160c85-cdd1-4bac-800a-9bb623b816d1', 'typeName': 'Report'}], ...() }`

File: metadata_service/proxy/atlas_proxy.py [hunk 2]
```diff
@@ -344,6 +345,30 @@ def _serialize_columns(self, *, entity: EntityUniqueAttribute) -> \
             )
         return sorted(columns, key=lambda item: item.sort_order)
 
+    def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
+        reports = []
+        if guids:
+            report_entities_collection = self._driver.entity_bulk(guid=guids)
+            for report_entities in report_entities_collection:
```
Comment (mgorsk1) [inline metadata_service/proxy/atlas_proxy.py:352]: you can quickly expand with https://github.com/verdan/pyatlasclient/pull/2 (available in pyatlasclient 1.0.4)

**Comment by mgorsk1** [comment]
LGTM (just one nit)

wdyt @verdan ?

File: metadata_service/proxy/atlas_proxy.py (hunk 2 above)
Comment (dechoma) [reply metadata_service/proxy/atlas_proxy.py]: yep , i fixed it adbe31f

**Comment by feng-tao** [comment]
cc @verdan 
also could you bump the dependency?

**Comment by feng-tao** [comment]
I will update the version later actually. Thanks for the pr and review :)

**Pull request merged**